tests/spec/w3c/headers-spec.js

```diff
@@ -1962,9 +1962,6 @@ describe("W3C — Headers", () => {
       <section id="first-sub-section">
         <h3>Testing</h3>
       </section>
-      <p id="p3">
```

Comment:
The structuring code doesn't like this kind of thing... it makes a new section or something... this kind of thing shouldn't happen anyway, so 🤷‍♂️

Reply:
I think this should be wrapped into a section of its own?

Reply:
Section without heading that the linter shouts at.

Reply:
I think that gets messy/confusing, tbh... plus it's too much of an edge case to worry about.

Reply:
Let's skip it then